src/components/Calendar/CourseCalendarEvent.tsx

```diff
@@ -167,14 +172,18 @@ const CourseCalendarEvent = (props: CourseCalendarEventProps) => {
                         </tr>
                         <tr>
                             <td className={classes.alignToTop}>Location</td>
-                            <td className={`${classes.multiline} ${classes.rightCells}`}>
-                                {bldg !== 'TBA' ? (
-                                    <a href={genMapLink(bldg)} target="_blank" rel="noopener noreferrer">
-                                        {bldg}
-                                    </a>
-                                ) : (
-                                    bldg
-                                )}
+                            {/* eslint-disable-next-line jsx-a11y/no-noninteractive-element-interactions */}
```

Comment:
Can you add another comment (in code) explaining why we need this workaround?

Reply:
I think the best solution to this is to add an `onClick` event handler to the original link. You can use `event.preventDefault` to prevent the browser from following the link, and then add your custom logic. Example

```tsx
<td>
   <a onClick={e => {
       e.preventDefault();
       selectBuilding({ location: bldg, courseName: title}}
    }}>
   </a>
</td>
```

You're not supposed to add `onClick` handlers to non-interactive elements, e.g. divs, spans, etc. and usually the solution is to use an interactive element and modify its behavior or appearance.